test/manifolds/probability_simplex.jl

```diff
@@ -60,6 +62,24 @@ include("../utils.jl")
                 retraction_methods=[SoftmaxRetraction()],
                 test_inplace=true,
                 vector_transport_methods=[ParallelTransport()],
+                test_rand_point=true,
+                test_rand_tvector=true,
+                rand_tvector_atol_multiplier=10.0,
```

Comment:
```suggestion
                rand_tvector_atol_multiplier=20.0,
```
Almost G2G, this needs a slightly larger tolerance :slightly_smiling_face: